Cargo.toml

```diff
@@ -18,22 +17,20 @@ members = [
 
 [workspace.package]
 edition = "2021"
-version = "0.0.1"
```

Comment:
I'd be inclined to keep using this - it's very annoying managing multiple version numbers.

That is, once/if we're publishing `vello_shaders` because we've done #467, we'd want it have the same version number as `vello` itself.
I don't think we're likely to have a breaking release of Vello which isn't also breaking the encoding

Reply:
If we have these smaller packages that are meant to be used by others than Vello, then publishing a breaking change for them when it isn't actually required would be creating a needless hurdle for those other consumers. Even if that isn't a common scenario.

There is also no sense in publishing new versions without any actual changes. So take for example a scenario where `vello` needs a small bug fix and we publish `0.1.1`. Well the encoding/shaders didn't actually receive any changes.

The opposite case is also relevant. Say we make two small patch releases for shaders, going from `0.1.0` to `0.1.2`. However `vello` proper might not need a new release at all.

There is also no way to specify dependencies with the workspace version. That is, under the `[dependencies]` table we still need to manually enter a version requirement for encoding and shaders. This isn't required for unpublished packages, but we're moving into the published state now.

Given all of the above, I consider the extra annoyance of having to bump the encoding/shader package's version field worth the extra flexibility and precision we get.

Reply:
> The opposite case is also relevant. Say we make two small patch releases for shaders, going from 0.1.0 to 0.1.2. However vello proper might not need a new release at all.

We probably want to be able to have a release of Vello which definitely has these changes, if we have improved the shaders. I don't want to be recommending that people depend on `vello_shaders = 0.1.2` to guarantee their users actually get the fix the need. That is, `vello_shaders` being updated is sufficient reason for `vello` to be released.

I don't think slightly optimising for the experience of the (very few) consumers who are going to be using `vello_shaders` directly, without using `vello`, at the expense of making things slightly more confusing for those many more who will be using `vello` directly is a good tradeoff. And I think it makes our processes more complicated, to maintain version numbers for each of the independent crates.

And if you're using `vello_encoding`, you want to be using the matching version of `vello` anyway. Having to match those up would make having the dependency much harder.

Reply:
I'm not fully sold, but this topic can be postponed until we're actually doing more of these releases. So for now, in the interest of getting other stuff here merged, I'll just revert the to-be-published packages to a unified version with some comments.

The dependencies will still need manual updates, which I'll also add a comment for.